src/lists.c

```diff
@@ -77,14 +77,14 @@ static Int IsListObject(Obj obj)
 **  This is, in some sense, a workaround for the not yet implemented features
 **  below (see LENGTH).
 */
-Int             (*IsSmallListFuncs [LAST_REAL_TNUM+1]) ( Obj obj );
+BOOL (*IsSmallListFuncs[LAST_REAL_TNUM + 1])(Obj obj);
 
 static Obj IsSmallListFilt;
 static Obj HasIsSmallListFilt;
 static Obj LengthAttr;
 static Obj SetIsSmallList;
 
-static Int IsSmallListObject(Obj obj)
+static BOOL IsSmallListObject(Obj obj)
```

Comment:
Same as above, some return values are still `1` and `0`.

Reply:
I changed this now, too, but as explained above, I don't really want to change all such places in this PR; I'd rather leave this to a follow up PR (or multiple)